config/fxdata/creature.cfg

```diff
@@ -605,7 +605,7 @@ RangeMin = MIN
 RangeMax = 284
 PrimaryTarget = 3
 Properties = MELEE_ATTACK
-Function = creature_fart 0 0
```

Comment:
this line should not change so it doesn't break Fart if a config has made a [shot33]

Reply:
That would defeat the purpose of this PR.

Reply:
Yes, I do believe we have to accept that. Sometimes we need new shots.